app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Users::SessionsController < Devise::SessionsController
+  # before_action :configure_sign_in_params, only: [:create]
+
+  # GET /resource/sign_in
+  # def new
+  #   super
+  # end
+
+  # POST /resource/sign_in
+  # def create
+  #   super
+  # end
+
+  # DELETE /resource/sign_out
```

Comment:
- [ ] At the moment I am unable to sign out of the application. As such I cannot create new users or try sign in again with the first user account I created. Please review this.
![image](https://github.com/obed300/budget-app/assets/62849872/04b9641a-6f43-4225-8e04-c853c65899c2)